apps/algorithm/search/src/routes.py

```diff
@@ -29,8 +29,8 @@
 @search_router.get("/api/search")
 async def search(
     query: str,
-    latitude: float,
-    longitude: float,
+    lat: float,
+    lon: float,
```

Comment:
Is this api spec compatible?

Reply:
I don't believe so, the lat and lon fields only need to be named that in elasticsearch specifically (not for our search endpoint). This change should be reverted I believe